torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -3,43 +3,41 @@
 namespace torch {
 namespace distributed {
 namespace rpc {
-namespace {
-  py::object module_;
-  py::object runUDFFunction_;
-  py::object loadResultFunction_;
-} // anonymous namespace
 
-namespace PythonRpcHandler {
-  void init() {
-    AutoGIL ag;
-    if (module_ == nullptr) {
-      module_ = py::module::import("torch.distributed.internal_rpc_utils");
-    }
-    if (runUDFFunction_ == nullptr) {
-      runUDFFunction_ = module_.attr("run_python_udf_internal");
-    }
-    if (loadResultFunction_ == nullptr) {
-      loadResultFunction_ = module_.attr("load_python_udf_result_internal");
-    }
+PythonRpcHandler::PythonRpcHandler() {
+  AutoGIL ag;
+  py::object module =
+      py::module::import("torch.distributed.internal_rpc_utils");
+  if (runUDFFunction_ == nullptr) {
```

Comment:
We don't need these if statements in the constructor since this will happen only once.